packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts

```diff
@@ -700,23 +700,15 @@ function _validateFees(totalBuyerFeeBasisPoints: number, totalSellerFeeBasisPoin
 function _getBuyFeeParameters(
   totalBuyerFeeBasisPoints: number,
   totalSellerFeeBasisPoints: number,
-  sellOrder?: UnhashedOrder,
+  sellOrder?: RawOrder,
   network?: 'mainnet' | 'rinkeby'
 ) {
   _validateFees(totalBuyerFeeBasisPoints, totalSellerFeeBasisPoints)
   let makerRelayerFee
   let takerRelayerFee
   if (sellOrder) {
-    // Use the sell order's fees to ensure compatiblity and force the order
-    // to only be acceptable by the sell order maker.
-    // Swap maker/taker depending on whether it's an English auction (taker)
-    // TODO add extraBountyBasisPoints when making bidder bounties
-    makerRelayerFee = sellOrder.waitingForBestCounterOrder
-      ? new BigNumber(sellOrder.makerRelayerFee)
-      : new BigNumber(sellOrder.takerRelayerFee)
-    takerRelayerFee = sellOrder.waitingForBestCounterOrder
-      ? new BigNumber(sellOrder.takerRelayerFee)
-      : new BigNumber(sellOrder.makerRelayerFee)
+    makerRelayerFee = new BigNumber(sellOrder.maker_relayer_fee)
+    takerRelayerFee = new BigNumber(sellOrder.taker_relayer_fee)
```

Comment:
I don't understand why this was changed

Reply:
I'm still trying to figure this out, but it seems to work for all orders (make offer, bid and buy)... suspiciously...

Reply:
I'd rather not change it unless it was breaking something and if it was breaking then let's chat about it on slack